spec/models/spree/order_spec.rb

```diff
@@ -259,11 +259,12 @@
     let(:shipment) { create(:shipment_with, :shipping_method, shipping_method: shipping_method) }
     let(:order)          { create(:order, shipments: [shipment]) }
     let(:enterprise_fee) { create(:enterprise_fee) }
-    let!(:adjustment)    { create(:adjustment, adjustable: order, originator: enterprise_fee, label: "EF", amount: 123, included_tax: 2) }
 
     before do
       Spree::Config.shipment_inc_vat = true
       Spree::Config.shipping_tax_rate = 0.25
+
+      adjustment = create(:adjustment, adjustable: order, originator: enterprise_fee, label: "EF", amount: 123, included_tax: 2)
```

Comment:
Do you need to assign it to the variable `adjustment`?

Reply:
ah, no. good point! I removed it. :+1: